src/types/GradeLevel.test.ts

```diff
@@ -0,0 +1,63 @@
+import GradeLevel from './GradeLevel';
+
+test('check Grade key values', () => {
+  const keys = [];
+  for (let k = 8; k <= 22; k++) keys.push(k);
+
+  expect(GradeLevel.keys()).toEqual(keys);
+});
+
+test('toString() - middle school', () => {
```

Comment:
These `toString()` tests aren't particularly interesting because they all test the same thing: that a value from a mapping gets used. Also after high school the grade level values become a bit hard to decipher. You should use `GradeLevel` enums themselves.

More interesting test cases would be if there were some sort of `rangeToString()` function that can simplify the string represenation of multiple values. E.g. `High School Freshman, Sophomore, Junior` if those three are selected or `All High School students` if all four High School options are selected.

Reply:
interesting, I can come back to this in the near future.